src/snowflake/snowpark/_internal/analyzer/metadata_utils.py

```diff
@@ -17,6 +17,13 @@
     from snowflake.snowpark._internal.analyzer.analyzer import Analyzer
 
 
+class DescribeQueryTelemetryField(Enum):
+    TYPE_DESCRIBE_QUERY_DETAILS = "snowpark_describe_query_details"
+    SQL_TEXT = "sql_text"
+    E2E_TIME = "e2e_time"
+    STACK_TRACE = "stack_trace"
```

Comment:
It's good that we're capturing the stack trace. But I imagine we'll need to further process it to extract the specific operations resulting in describe queries?

Reply:
yea, we can do the parsing on the dashboard side